src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/controllers/CaseActionsForCaseWorkerController.java

```diff
@@ -264,6 +267,10 @@ public ResponseEntity<CCDCallbackResponse> postDefaultValues(
             FlagsImageHelper.buildFlagsImageFileName(ccdRequest.getCaseDetails());
             caseData.setMultipleFlag(caseData.getEcmCaseType() != null
                     && caseData.getEcmCaseType().equals(MULTIPLE_CASE_TYPE) ? YES : NO);
+
+            if (ET1_ONLINE_CASE_SOURCE.equals(caseData.getCaseSource())) {
```

Comment:
Wasn't this in last release? :O I remember it was being tested and passed

Reply:
discussed with Gaurav on Slack - code was correct but needed to be moved